v19.2/use-the-built-in-sql-client.md

```diff
@@ -740,6 +741,61 @@ Time: 2.426534ms
 > SHOW DATABASE
 ~~~
 
+### Repeat a SQL statement
+
+Use the `--execute` flag to create a table from the command line:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure \
+--execute="CREATE TABLE t1 (id UUID DEFAULT gen_random_uuid(), name STRING)"
+~~~
+
+With the `--watch` flag, you can repeat the statements specified with the `--execute` flag periodically until an error occurs:
```

Comment:
I would be more explicit here about what we mean by "an error occurs".  For example, do we mean: SQL error?  UNIX process kill?  Both/either?  If both/either, maybe some text like "until a SQL error occurs, or the process is killed.  In this example we kill the process by typing ctrl-c."  or something like that.  Also, I would include the text from the code PR that "this is intended for simple monitoring scenarios during development and testing", since the motivation for this added flag was not clear to me until I went to the CRDB commit.